README.md

```diff
@@ -57,16 +57,16 @@ the connector will return an error.
-| `url`                     | Connection string for the Postgres database.                                                                                               | true     |               |
-| `tables`                  | List of table names to read from, separated by comma. Example: `"employees,offices,payments"`. Using `*` will read from all public tables. | true     |               |
-| `snapshotMode`            | Whether or not the plugin will take a snapshot of the entire table before starting cdc mode (allowed values: `initial` or `never`).        | false    | `initial`     |
-| `cdcMode`                 | Determines the CDC mode (allowed values: `auto`, `logrepl` or `long_polling`).                                                             | false    | `auto`        |
-| `logrepl.publicationName` | Name of the publication to listen for WAL events.                                                                                          | false    | `conduitpub`  |
-| `logrepl.slotName`        | Name of the slot opened for replication events.                                                                                            | false    | `conduitslot` |
-| `logrepl.autoCleanup`     | Whether or not to cleanup the replication slot and pub when connector is deleted                                                                                            | false    | `true` |
-| ~~`table`~~               | List of table names to read from, separated by comma. **Deprecated: use `tables` instead.**                                                | false    |               |
+| name                      | description                                                                                                                                   | required | default       |
+|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| `url`                     | Connection string for the Postgres database.                                                                                                  | true     |               |
+| `tables`                  | List of table names to read from, separated by comma. Example: `"employees,offices,payments"`. Using `*` will read from all public tables.    | true     |               |
+| `snapshotMode`            | Whether or not the plugin will take a snapshot of the entire table before starting cdc mode (allowed values: `initial` or `never`).           | false    | `initial`     |
+| `cdcMode`                 | Determines the CDC mode (allowed values: `auto`, `logrepl`).                                                                                  | false    | `auto`        |
```

Comment:
Since we have only `logrepl` available, it looks like `cdcMode` is not needed anymore.

Reply:
@hariso that's my thought as well. However, do we want to keep it since we know we'll implement it?

Reply:
Hmm, I'd say it's the same situation as with long_polling: having it as a placeholder could be confusing.

Reply:
@hariso @lovromazgon my thinking is to _signal_ users that there will be more modes to come beyond what's default rather than to indicate this only will be meant for logical replication.